### PR TITLE
103 faq how do i find my session id in posit workbench

### LINKED ISSUE #49: REQ - How do I find out the Session ID of my Posit Workbench session?
**Is your feature request related to a problem? Please describe.**
The Posit Programme would like to capture greater detail about the issues that users are experiencing with Posit Workbench.  One such detail is the Session ID of the user's Posit Workbench session. It is not obvious to the user how to find out the Session ID.

**Describe what you'd like**
Clear and concise, step-by-step instructions that a Posit Workbench user can follow to find out the Session ID of their session.

**Describe alternatives you've considered**
There are no alternatives.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by JFix89**
# Pull Request Details

**Issue Number**: Closes #49 and #103 

**Type**: Documentation

## Description of the Change

Updated FAQs.md to add guidance on how to find a Posit Workbench session ID. This content had already been reviewed for issue #49 but not merged at the time. Instead of adding to a dedicated markdown file in the technical documentation, this content is now being added to the new 'Sessions' section of the FAQs file.

### Verification Process

N/A

### Additional Work Required

N/A

## Release Notes

Updated FAQs.md to add guidance on how to find a Posit Workbench session ID

### CODE REVIEW

File: Posit Infrastructure/FAQs.md [hunk 1]
```diff
@@ -56,6 +57,39 @@ Microsoft Edge puts your tabs to sleep when you're not using them. Tabs are put
 
 ### Sessions
 
+#### How do I find my session ID?
+
+##### Background
+
+When a Posit Workbench session is requsted by the user, it is assigned a session ID.  This is a long string of letters and numbers that uniquely identifies the session. The session ID corresponds to metrics and logs that are produced by the Posit Workbench application and collected for onward analysis and debugging of issues.
```
Comment (alasdairgm) [inline Posit Infrastructure/FAQs.md:64]: "requested" misspelled

File: Posit Infrastructure/FAQs.md [hunk 2]
```diff
@@ -56,6 +57,39 @@ Microsoft Edge puts your tabs to sleep when you're not using them. Tabs are put
...
+
+##### Steps to find out the session ID of your Posit Workbench session
+
+1. If you have not already done so, log into Posit Workbench and start a session (refer to the [guidance on the Knowledge base here](https://public-health-scotland.github.io/knowledge-base/docs/Posit%20Infrastructure?doc=How%20to%20Access%20Posit%20Workbench.md#logging-in) if necessary).
+
+2. After the session has started, navigate to the [Posit Workbench homepage](https://pwb.publichealthscotland.org) and identify the session you would like the session ID of in the list of sessions, as circled in red below:
```
Comment (alasdairgm) [inline Posit Infrastructure/FAQs.md:70]: maybe something like "identify the session for which you would like to obtain the session ID" for readability?

**Review by rmccreath — APPROVED** [review]
Looks good, happy to approve.

Just to add though, I think the wording was extended out as it was it's own document, so could potentially be reduced to something like intro text, point 2 and the alternative method?

File: Posit Infrastructure/FAQs.md (hunk 1 above)
Comment (JFix89) [reply Posit Infrastructure/FAQs.md]: Corrected in the latest commit!

**Comment by JFix89** [comment]
Thanks both for reviewing, I've made the changes to the text that @alasdairgm suggested. @rmccreath : Although I agree it's on the long side, I think it provides a good amount of detail for beginners, so I'm tempted to leave it as it is for now. Happy to revisit this at a later point if we need to cut down content in the FAQs.md file though!

**Pull request merged**